### PR TITLE
Made all crud for the timeline and post functionality

### PR DESCRIPTION
**Pull request by JensIssa**

### CODE REVIEW

**Comment by kasp441** [comment]
merge in from development first

**Comment by JensIssa** [comment]
OKAY DONE NOW REVIEW @kasp441

File: PostsService/Controllers/TimelineController.cs
Comment (kasp441) [inline PostsService/Controllers/TimelineController.cs:1]: needs different tags for reading from route or from body to be able to read inbound

**Review by kasp441 — CHANGES_REQUESTED** [review]
update controller to handle reading params from route

File: PostsService/Controllers/PostController.cs
Comment (kasp441) [inline PostsService/Controllers/PostController.cs:1]: make your final route more handle friendly.
from "api/controller/function" -> "/function" or "controller/function"

**Review by kasp441 — APPROVED** [review]
changes have been remedied. nice
approved

**Pull request merged**